trainee_template/react_series.amlg

```diff
@@ -447,6 +447,10 @@
 						))
 				))
 
+				(if (= (size existing_series_cases) 0)
+					(accum (assoc warnings (assoc "There is no series trained with the given set of IDs.")))
```

Comment:
What if the user gives a list of IDs in which some are in the Trainee and others are not? Maybe we should check for that and warn something like "One or more of the IDs are not present in the Trainee.." or something to that effect.

Reply:
This bit of logic is in like a per-series section. So "set of IDs" here refers to the set of IDs for a *single* series (There can be more than one ID feature).

I kind of want a good way to add the specific IDs that no cases re found for, but not sure the best way.

Reply:
I added some logic to print the specific IDs that do not have related trained cases. So this shouldn't be an issue.